client/test/account_store.spec.js

```diff
@@ -396,8 +396,26 @@ describe('AccountStore', () => {
 
 
     describe('Should REMOVE account(s)', () => {
-      it.skip('REMOVE_ACCOUNT_SUCCESS', () => {
-        // FIXME Nothing is done here in the store code.
+      beforeEach(() => {
```

Comment:
@aenario : shouldn't you reset accounts after each test? ie. https://github.com/aenario/cozy-emails/blob/41b0c037ddbdeb44c419ad039b0118cfade4827a/client/test/account_store.spec.js#L302